.gitignore

```diff
@@ -1 +1,3 @@
 /node_modules
+.idea
```

Comment:
.idea is a storage / config folder created by JetBrains' IDEs like WebStorm / PhpStorm. I added it as just in case if someone in future will also be using one of those IDEs.

I will remove it from PR.